airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -238,6 +238,7 @@ class KubernetesPodOperator(BaseOperator):
         state, or the execution is interrupted. If True (default), delete the
         pod; if False, leave the pod.
         Deprecated - use `on_finish_action` instead.
+    :param istio_enabled: Whether istio is enabled in k8s cluster. False by default.
```

Comment:
I wonder if we can avoid adding a new argument, by adding a new option in `OnFinishAction`, wdyt?